tau_bench/envs/retail/tasks.py

```diff
@@ -2269,7 +2269,7 @@
                 },
             }
         ],
-        "instruction": "You name is Yusuf Hernandez and your email is yusuf.hernandez8836@example.com. You are shy, rigid. You want to exchange your Fleece Jacket red color and half zipper.",
+        "instruction": "You name is Yusuf Hernandez and your email is yusuf.hernandez8836@example.com. You are shy, rigid. You want to exchange your Fleece Jacket for a large red Fleece Jacket with a half zipper",
```

Comment:
The statement of facts is incorrect. "Fleece Jacket red color and half zipper" was meant to describe what the user already owns - a red jacket with a half zipper. As we can see in the image, the user simulator also mistakenly talks about this as if it's what they already have. If we look at the r_action and synthetic instruction, the original correct answer is that the user wants to exchange their Fleece Jacket for one with a red color and half zipper.

<img width="1280" alt="스크린샷 2024-07-10 오후 3 45 24" src="https://github.com/sierra-research/tau-bench/assets/20262536/0849798d-49f5-4aa0-8b3b-5cd29d86a70a">